nss-day-cohort-30/bangazon-api-ambitious-albatrosses
# Product & Product Type Controllers & Tests

Fixes:
- N/A

Changes proposed in this request & reasons behind organizational or architectural decisions:
- Wrote controllers for product and product type resources
- Wrote tests for product and product type controllers

Steps to test:
- STEP 1) Run the SQL from [this file](https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/blob/master/AmbitiousAlbatrossesData.txt) in Azure Data Studio

- STEP 2) Open Visual Studio Test Explorer and locate the TestProducts and TestProductTypes folders within the TestBangazonAPI folder.

- STEP 3) Select and Run all tests within the folders.

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
- N/A

Commit message:
- tests pass for CRUD functionality for product and product type resources

Link to feature ticket:
- [#3](https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/issues/3)
- [#4](https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/issues/4)

@ambitions-albatrosses
